apps/projects/templates/a4projects/includes/project_list_tile.html

```diff
@@ -1,15 +1,36 @@
-{% load i18n project_tags thumbnail %}
+{% load i18n project_tags thumbnail static %}
 
 <div class="tile">
-    {% get_days project.days_left as days %}
-    {% if project.has_finished %}
-    <span class="label">{% trans "Finished" %}</span>
-    {% elif days %}
-    <span class="label">{{ days }}</span>
-    {% endif %}
+    <div class="tile__head">
+        <a href="{% url 'project-detail' project.slug %}">
+            <img src="http://placehold.it/285x170" alt="{{ project.name }}">
```

Comment:
The `alt` is redundant because `project.name` is also available from `list-item__title`